airbyte-ci/connectors/pipelines/pyproject.toml

```diff
@@ -29,4 +29,5 @@ pytest = "^6.2.5"
 pytest-mock = "^3.10.0"
 
 [tool.poetry.scripts]
-airbyte-ci = "pipelines.commands.airbyte_ci:airbyte_ci"
+airbyte-ci-bare = "pipelines.commands.airbyte_ci:airbyte_ci"
```

Comment:
Love that were brining in `dagger run`.

One question though why use `airbyte-ci-bare` and no `airbyte-ci --disable-dagger-run  ...`?

Seems like that is possible and better ergonomics for users.

Reply:
rewording: We can indeed have `airbyte-ci --disable-dagger-run`  easily. But I'm not sure we can easily get rid of a second executable. The `airbyte-ci-bare` use won't be documented...

Reply:
@bnchrch I'll try some things out to check how we could not rely on a second executable.

Reply:
I renamed `airbyte-ci-bare` to `airbyte-ci-internal`. 
You can use `airbyte-ci --no-tui` to disable the terminal UI (and directly use `airbyte-ci-internal`).

Reply:
@bnchrch I'll keep running `airbyte-ci-internal` in the GHA action to prevent any regression